runtime/src/bank/tests.rs

```diff
@@ -6932,6 +7048,12 @@ fn test_bpf_loader_upgradeable_deploy_with_max_len() {
         assert!(slot_versions.is_empty());
     }
 
+    // Advance bank to get a new last blockhash for duplicate transaction below
```

Comment:
It's because this test used to assume that when you invoke a program that doesn't exist, the transaction wouldn't get committed. So the test reused the same `invocation_message` and blockhash to test that invoking the program _now_ works after the program exists. After this PR, if you invoke a program that doesn't exist, the transaction _will_ get committed still if the fee payer is valid. So later we need to use a new blockhash when creating what is otherwise the exact same transaction to test invocation a second time. I'll add a comment in the test.